contents/tutorials/performance-metrics.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Understanding page load time metrics in performance monitoring
+sidebar: Docs
+showTitle: true
+author: ['lior-neu-ner']
+date: 2023-04-12
+featuredImage: todo
+tags: []
+---
+
+Waiting for slow websites is like watching paint dry. It's the bane of productivity, the destroyer of efficiency, and the enemy of progress.
```

Comment:
Perhaps, instead of "enemy of progress", the final point could be a "a leading cause of customer churn"? There must be some data somewhere to support that and it's more relevant to users.

Also, I'd probably say "slow web apps" rather than websites given our ICP.

Reply:
Good advice, thanks!